### PR TITLE
Multiselect rework

### PR DESCRIPTION
**Pull request by AlphaLima**
Complete rework of mutliselect for:
-Flarm
-Oracle
-Team
-Own Position
alternative threadsafe UTF8 Pictorials 
all changes with Option switch in Options.h
Map Pictorial as UTF8 or Bitmap switchable during runtime by  
Appearance.UTF8Pictorials = UTF8PICTORIALS;
Fallback to Bitmap for non 64Bit Windows Systems

decreased the search area on big zoom scales for a more precise feedback.
disabled UTF8 pictorialf by  default

Example:
multisel
multisel

### CODE REVIEW

File: Common/Source/Globals.cpp [hunk 1]
```diff
@@ -17,9 +17,47 @@
...
+      {
+          //handle error
+      }
+  }
+  return bIsWow64;
+#else
+  return UTF8PICTORIALS;
+#endif
+}
+
```
Comment (brunotl) [inline Common/Source/Globals.cpp]: don't work for PCX64 target

File: Common/Source/Dialogs/dlgMultiSelectList.cpp [hunk 2]
```diff
@@ -279,175 +534,165 @@ static void OnMultiSelectListPaintListItem(WindowControl * Sender, LKSurface& Su
...
+#ifdef TEAM_CODE_MS
+            /************************************************************************************************
+             * IM_TEAM
+             ************************************************************************************************/
+        case IM_TEAM:
+          _sntprintf(text1,MAX_LEN,_T("%s:"),gettext(_T("_@M700_"))); //_@M700_ "Team code"
+          _sntprintf(text2,MAX_LEN,_T("%s"), CALCULATED_INFO.OwnTeamCode );
+            ShowTextEntries(Surface, rc,  text1, text2);
+            if(Appearance.UTF8Pictorials)
+              UTF8Pictorial( Surface,  rc, _T("⚑"), RGB_VDARKRED);
```
Comment (brunotl) [inline Common/Source/Dialogs/dlgMultiSelectList.cpp]: string literal containing non usascii characters are not crossplatform safe, neet bo be moved in language file

File: Common/Source/Draw/DrawFlightMode.cpp [hunk 3]
```diff
@@ -13,6 +13,94 @@
...
+        Col = RGB_BLACK;
+      break;
+    }
+#ifdef KOBO
+    Col = RGB_LIGHTGREY;
+#endif
+
+    return Col;
+}
+
```
Comment (brunotl) [inline Common/Source/Draw/DrawFlightMode.cpp]: string literal containing non usascii characters are not crossplatform safe, neet bo be moved in language file

File: Common/Source/Draw/LKDrawWaypoints.cpp [hunk 4]
```diff
@@ -694,8 +708,167 @@ void MapWindow::DrawWaypointPictoBg(LKSurface& Surface, const RECT& rc) {
...
+      _stprintf(pPict, _T("✈"));
+      Col = LKColor(199,21,133);
+    break;
+  } // switch estyle
+#ifdef KOBO
+  Col = LKColor(30,30,30);
+#endif
+return Col;
+}
+
```
Comment (brunotl) [inline Common/Source/Draw/LKDrawWaypoints.cpp]: string literal containing non usascii characters are not crossplatform safe, neet bo be moved in language file

File: Common/Source/Globals.cpp (hunk 1 above)
Comment (brunotl) [reply Common/Source/Globals.cpp]: with IsWow64Process, you check if process is 32bit windows applications runing on 64bit windows, not a 64bit process running on 64bit windows

File: Common/Source/LKProfileResetDefault.cpp [hunk 5]
```diff
@@ -266,6 +266,7 @@ void LKProfileResetDefault(void) {
   InverseInfoBox_Config=true; // black
   Appearance.InfoBoxModel=apImPnaGeneric;
 
+  Appearance.UTF8Pictorials = UTF8PICTORIALS;
```
Comment (brunotl) [inline Common/Source/LKProfileResetDefault.cpp]: not same as Global_Init() ?

File: Common/Source/Globals.cpp (hunk 1 above)
Comment (AlphaLima) [reply Common/Source/Globals.cpp]: OK, what do you recomend to do? 
(Keep in mind, it is configurable anyway. Default is the Bitmaps not UTF8)

File: Common/Source/LKProfileResetDefault.cpp (hunk 5 above)
Comment (AlphaLima) [reply Common/Source/LKProfileResetDefault.cpp]: Will change it, already prepared a rebased version after merge of NotamMultiselect.

**Comment by AlphaLima** [comment]
replaced by #1230

**Pull request closed without merging**